mmcv/parallel/scatter_gather.py

```diff
@@ -43,17 +47,22 @@ def scatter_map(obj):
     try:
         return scatter_map(inputs)
     finally:
-        scatter_map = None
+        scatter_map = None  # type: ignore
 
 
-def scatter_kwargs(inputs, kwargs, target_gpus, dim=0):
+def scatter_kwargs(inputs: ScatterInputs,
+                   kwargs: ScatterInputs,
```

Comment:
`ScatterInputs` is not appropriate because `input` should not be dict and `kwargs` also should not be list or tuple.

Reply:
How should I modify it?